src/software/visualizer/drawing/navigator.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include "software/ai/navigator/navigator.h"
-#include "software/gui/drawing/draw_functions.h"
+#include "software/visualizer/drawing/draw_functions.h"
 
 // TODO: comment
```

Comment:
I'm going to preemptively say this `TODO` was from before this PR and is being addressed in a separate set of changes I'm working on.